lib/ansible/modules/remote_management/manageiq/manageiq_provider.py

```diff
@@ -181,6 +181,19 @@
         required: false
         default: null
         description: The CA bundle string with custom certificates. defaults to None.
+
+  auth_type:
+    required: false
+    description: Auth type endpoint configuration information. E.g. RSA key pair
+    default: null
+    version_added: "2.5"
+    suboptions:
+      userid:
+        description: User ID for auth type.
```

Comment:
Sorry, what does "for auth type" mean here?
I suspect this is "Username for SSH login into hosts"?

Reply:
It is, have updated.